drivers/wifi/simplelink/simplelink_support.c

```diff
@@ -115,10 +122,57 @@ static s32_t configure_simplelink(void)
 	retval = sl_WlanProfileDel(0xFF);
 	ASSERT_ON_ERROR(retval, WLAN_ERROR);
 
+#if defined(CONFIG_NET_IPV4) && defined(CONFIG_NET_CONFIG_MY_IPV4_ADDR)
+	if (net_addr_pton(AF_INET, CONFIG_NET_CONFIG_MY_IPV4_ADDR, &addr4)
+			< 0) {
+		LOG_ERR("Please verify CONFIG_NET_CONFIG_MY_IPV4_ADDR");
```

Comment:
Minor, but I'd say, consistent Zephyr error style would be "Invalid CONFIG_NET_CONFIG_MY_IPV4_ADDR".

Reply:
Thanks, done.